packages/location-field/i18n/en-US.yml

```
@@ -26,6 +26,8 @@ otpUi:
     # Note to translator: This is an implicit plural (as in "Other results").
     other: Other
     recentlySearched: Recently Searched
+    suggestedLocations: Suggested locations
+    suggestedLocationsLong: Open the list of {locationType} location suggestions
```

Let's not use `locationType` in this message, or we would need to add the different values to the languages files.
```suggestion
    suggestedLocationsLong: Open the list of suggested locations
```

---

Hmm, we might need a "Close list of suggested locations" too...

---

"toggle"?

---

Sure